packages/@react-native-windows/telemetry/src/telemetry.ts

```diff
@@ -140,156 +150,133 @@ export class Telemetry {
+    return false;
+  }
+
+  // The second telemetry processor that is executed after calling flush().
+  // Performs sanitization and PII removal from ExceptionData,
+  // applicable only to error scenarios.
+  private static errorTelemetryInitializer(envelope: coreOneDS.ITelemetryItem) : boolean
+  {
+    if (envelope.data?.baseType === 'ExceptionData') {
+      const exceptionData = envelope.data?.exceptionData;
+      if (exceptionData) {
+          for (const frame of exceptionData.parsedStack) {
+            errorUtils.sanitizeErrorStackFrame(frame);
+          }
+
+          // Exception message must never be blank, or AI will reject it
+          exceptionData.message = exceptionData.message || '[None]';
 
-    // Allow overriding the proxy server via an environment variable
-    const proxyServer = process.env[ENV_PROXY_OVERRIDE];
-    if (proxyServer) {
-      Telemetry.client.config.proxyHttpUrl = proxyServer;
-      Telemetry.client.config.proxyHttpsUrl = proxyServer;
+          // CodedError has non-PII information in its 'type' member, plus optionally some more info in its 'data'.
+          // The message may contain PII information. This can be sanitized, but for now delete it.
+          if (Telemetry.options.preserveErrorMessages) {
+            exceptionData.message = errorUtils.sanitizeErrorMessage(
+              exceptionData.message,
+            );
+          } else {
+            exceptionData.message = '[Removed]';
+          }
+      }
     }
 
-    Telemetry.client.config.disableAppInsights = Telemetry.isTest;
-    Telemetry.client.config.disableStatsbeat = true;
+    return true;
+  }
 
-    // Despite trying to disable the statsbeat, it might still be running: https://github.com/microsoft/ApplicationInsights-node.js/issues/943
-    // So we want to disable it, and despite the method's typing, getStatsbeat() _can_ return undefined
+  private static addTelemetryInitializers()
+  {
+    Telemetry.appInsightsCore?.addTelemetryInitializer(Telemetry.basicTelemetryInitializer);
+    Telemetry.appInsightsCore?.addTelemetryInitializer(Telemetry.errorTelemetryInitializer);
+  }
 
-    Telemetry.client.getStatsbeat()?.enable(false);
+  /** Sets up Telemetry.appInsightsCore. */
+  private static setupClient() {
+    const postChannel: PostChannel = new PostChannel();
+
+    const coreConfiguration: coreOneDS.IExtendedConfiguration = {
+      instrumentationKey: process.env[ENV_SETUP_OVERRIDE] ?? RNW_1DS_INSTURMENTATION_KEY,
+    }
+
+    const postChannelConfig: IChannelConfiguration = {
+      eventsLimitInMem: 5000
+    };
 
-    Telemetry.client.channel.setUseDiskRetryCaching(!Telemetry.isTest);
+    coreConfiguration.extensionConfig = {};
+    coreConfiguration.extensionConfig[postChannel.identifier] = postChannelConfig;
+
+    // Allow overriding the endpoint URL via an environment variable.
+    if (process.env[ENV_PROXY_OVERRIDE] !== undefined) {
+      coreConfiguration.endpointUrl = process.env[ENV_PROXY_OVERRIDE];
```

Comment:
I don't think endpointUrl is strictly the same functionality here. The purpose of the proxy override is to allow local network sniffing and testing with tools like Fiddler which act as MITM proxy servers.

Please check if there's an exposure or method for specifying an HTTP(s) proxy server for the telemetry submission.

Reply:
There is no API in 1DS resembling proxy stuff. This endpointUrl prop is what seemed to be the closest resembling API. I sent an email to the thread with AI to double check this.

Reply:
Ok so we're currently using the right API to change the URL endpoint, but it doesn't allow MITM (if that means to submit the data to both the custom URL and Aria).

Reply:
Got it. I'm not optimistic to finish this before I go OOF, so I filed an issue to track this: #13682